packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTileView/CatalogTileView.test.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { CatalogTileView, CatalogTileViewCategory } from './index';
+import { CatalogTile, CatalogTileBadge } from '../CatalogTile/index';
+
+const pfBrand = '<PatternFly Brand Image here>';
+const github = '<GitHub Brand Image here>';
+
+test('CatalogTile renders properly', () => {
+  const component = mount(
+    <CatalogTileView id="test-catalog-tile-view">
+      <CatalogTileViewCategory id="category-1" title="Category 1" totalItems={4} viewAll={false}>
+        <CatalogTile
+          featured
+          iconImg={pfBrand}
+          badges={[<CatalogTile.Badge type="fa" name="cog" title="Certified" id="certified" />]}
+          title="Patternfly"
+          vendor={
+            <span>
+              PatternFly is a <a href="redhat.com">Red Hat</a> sponsored project.
+            </span>
+          }
+          description="A community of designers and developers collaborating to build a UI framework for enterprise web applications."
+        />
+        <CatalogTile
+          iconImg={github}
+          badges={[
+            <CatalogTileBadge type="fa" name="cog" title="Certified" id="certified" />,
+            <CatalogTileBadge type="pf" name="ok" title="USDA Approved" id="approved" />
+          ]}
+          title="GitHub Desktop"
+          vendor="provided by GitHub"
+          description="Simple collaboration from your desktop."
+        />
+        <CatalogTile
+          featured
+          iconImg={pfBrand}
+          badges={[<CatalogTile.Badge type="fa" name="cog" title="Certified" id="certified" />]}
+          title="Patternfly"
+          vendor={
```

Comment:
Probably not worth changing at this point but FWIW Sam/Openshift prefer the term 'provider'.